src/main/java/com/sprint/mission/discodeit/entity/UserEntity.java

```diff
@@ -31,8 +32,8 @@ public class UserEntity extends BaseUpdatableEntity {
   @JoinColumn(name = "profile_id", unique = true)
   private BinaryContentEntity profileId;
 
-  @OneToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL, mappedBy = "user", optional = false)
-  private UserStatusEntity userStatus;
+  @Column(nullable = false)
+  private Role role = Role.USER;
```

Comment:
`@Enumerated(EnumType.STRING)` 애노테이션 안붙이면 JPA가 테이블 변경 시 smallint 타입으로 컬럼을 생성합니다!
```suggestion
  @Column(nullable = false)
  @Enumerated(EnumType.STRING)
  private Role role = Role.USER;
```